ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-ocp-ci-integration.yaml

```diff
@@ -0,0 +1,30 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: common-templates-builder
+    namespace: ci
+    tag: v0.1.0
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
+tests:
+- as: e2e-tests
+  commands: export TARGET=fedora && make e2e-tests
+  openshift_installer_custom_test_image:
+    cluster_profile: azure4
+    from: src
```

Comment:
Like written, this won't work unfortunately - `src` is not actually a custom image. I think this needs to be:

```
openshift_installer_src:
  cluster_profile: azure4
```
(no `from:`)

Second, this is adding a new template-based job, which we discourage at this point. This should be doable with multi-stage tests (see the [doc](https://steps.ci.openshift.org/help#config)). You would need to use `ipi-azure` workflow (https://steps.ci.openshift.org/workflow/ipi-azure), and override its `test` (see the third example in [doc](https://steps.ci.openshift.org/help#config)) with your step, Your test step would have `from: src`.

The exception to this is if your tests currently need the openshift cli (`oc`) - that is currently more convenient to achieve with templates.

Reply:
this tests needs `oc`, because it will create virtual machines from templates.

Reply:
Then it's fine to use templates.